src/commonMain/kotlin/fr/acinq/lightning/channel/states/Normal.kt

```diff
@@ -664,7 +664,11 @@ data class Normal(
                 sigStash = emptyList()
                 Pair(Offline(this@Normal.copy(spliceStatus = spliceStatus1)), failedHtlcs)
             }
-            else -> unhandled(cmd)
```

Comment:
I'm a bit afraid of this `else` in channel states, we're never sure what events we're actually ignoring and if they're safe to ignore. I only updated the `Normal` state, should I do the same thing in other states? We will end up with quite a few `is XXX -> unhandled(cmd)`, but at least it's explicit and we can then detect that a command is incorrectly ignored. This could be particularly useful when adding new `ChannelCommand`s, to force us to think about each channel state and whether it should handle that new command or not. WDYT?

Reply:
It makes adding new commands more involved, but is safer, and that doesn't happen often anyway. Let's try it, we can always revisit if needed.